ocr_performance/google_tesseract_transcripts.py

```diff
@@ -71,35 +73,40 @@ async def transcribe(page_image_file, ocr, trans_dir):
 
 
 async def transcribe_all(engine, trans_dir):
-    '''
+    """
     Scans through the image files in PHOTOS_DIR and invokes the
     transcribe() function on each
-    '''
+    """
     image_files = [x for x in listdir(PHOTOS_DIR) if x.endswith('.jpg')]
     image_files.sort()
 
     for idx, image_file in enumerate(image_files):
         print(f'Processing image<{idx}:{image_file}>')
         await transcribe(PHOTOS_DIR+image_file, ocr=engine, trans_dir=trans_dir)
 
-if __name__ == '__main__':
+    if __name__ == '__main__':
 
-    parser = ArgumentParser()
-    parser.add_argument('-e', '--engine', choices=[GOOGLE, TESS],
-                        default=TESS,
-                        help='Specify transcribe engine')
-    parser.add_argument('-l', '--language', choices=['ssq', 'storysquad'],
-                        default='storysquad',
-                        help='Specify language for Tesseract engine')
+        parser = ArgumentParser()
+        parser.add_argument('-e', '--engine',
+                            choices=[GOOGLE, TESS],
+                            default=TESS,
+                            help='Specify transcribe engine')
+        parser.add_argument('-l', '--language',
+                            choices=['ssq', 'storysquad'],
+                            default='storysquad',
+                            help='Specify language for Tesseract engine')
 
-    args = parser.parse_args()
+        args = parser.parse_args()
 
-    trans_dir = DATA_DIR + args.engine
-    if args.engine == GOOGLE:
-        engine = GoogleAPI()
-    else:
+        trans_dir = DATA_DIR + args.engine
+
+    if args.engine == TesseractAPI(args, language):
```

Comment:
Since you have language in your args on 99, you don't need to parameterize it as it will throw an unresolved reference error.